.github/workflows/benchmark.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       - name: Generate filters
         id: filter-setup
         run: |
-          filters=$(find . -maxdepth 1 -type d ! -path ./.git ! -path . -exec basename {} \; | grep -v '^\.' | awk '{printf "%s: \"%s/**\"\n", $1, $1}')
+          filters=$(find . -maxdepth 1 -type d ! -path ./.git ! -path . ! -path ./testhelpers -exec basename {} \; | grep -v '^\.' | awk '{printf "%s: \"%s/**\"\n", $1, $1}')
```

Comment:
question: the helper tests won't be executed because of this. Have you considered moving to `go.work` so that it's easier to identify the module paths?